TypeCobol/Compiler/CodeElements/CodeElement.cs

```diff
@@ -1,5 +1,5 @@
-using Antlr4.Runtime;
 using System;
+using Antlr4.Runtime;
```

Comment:
In the end, it appears this file hasn't been modified so could you revert to the original file from develop.

Reply:
File TypeCobol/Compiler/CodeElements/CodeElement.cs is identical identical from develop branch one
https://github.com/TypeCobolTeam/TypeCobol/pull/1600/commits/7b1c8285f6ae818069349c70ab185fa012fdd39a